.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Package and release
+
+# we need to let GitHub know _when_ we want to release, typically only when we create a new tag.
+# this will target only tags
+on:
+  push:
+    tags:
+      - '**'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    # specify the environment variables used by the packager, matching the secrets from the project on GitHub
+    env:
+      GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}  # "GITHUB_TOKEN" is a secret always provided to the workflow
+
+
+    # "steps" holds a list of all the steps needed to package and release our AddOn
+    steps:
+
+      # we first have to clone the AddOn project, this is a required step
+      - name: Clone project
+        uses: actions/checkout@v3
+
```

Comment:
Can you add a step here to remove the stuff from the repo that we don't want to get packaged up with the mod? Those files would be: `.github/` and `.luacheckrc` - not that they are sensitive data, but they're not relevant to the player

Something like:

```
- name: delete non-release files
- run: rm -rf .github/ .luacheckrc
```

It looks like this would also be achievable by adding a `.pkgmeta` file to the repo per https://github.com/BigWigsMods/packager/wiki/Preparing-the-PackageMeta-File#changing-the-package-name

Reply:
> Please look here: https://github.com/MyGamesDevelopmentAcc/sArena/releases/tag/testrelease_1 Those files do not land in the release already.
> 

Nice. Don't hesitate to share that kind of context (link to test release) in the PR description :)

> For curse to work it would require just adding the curse key by @stako .